app_basics/commands/validators.py

```diff
@@ -0,0 +1,36 @@
+from typing import Tuple
+# import .constants as con
+from .constants import constants as con
+
+
+# raw_cmd = '//quit' # input from user
+# // EXIT_CMD indexing
+def is_command_valid(raw_cmd: str) -> Tuple[str, bool]:
```

Comment:
bonus - think about how you can remodel the commands to make it easier to find a single command by either key or value